homeworks/src/main/java/hw39/Order.java

```diff
@@ -0,0 +1,24 @@
+package hw39;
+
+import lombok.*;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@NoArgsConstructor
+@Getter
+@Setter
+@EqualsAndHashCode
+@ToString
+public class Order {
```

Comment:
How does Hibernate know about the mapping, if neither `Order` nor `Product` are mapped as entities? I doubt you've tested this. It shouldn't work properly without entity mapping,